src/go/privileged-service/pkg/manage/install.go

```diff
@@ -46,9 +45,12 @@ func InstallService(name, displayName, desc string) error {
 		return err
 	}
 	defer m.Disconnect()
+
+	UninstallService(name)
+
 	s, err := m.CreateService(name, instPath, mgr.Config{DisplayName: displayName, Description: desc})
-	if errors.Is(err, windows.ERROR_SERVICE_EXISTS) {
-		return errors.Wrapf(err, "service [%s] already exists", name)
+	if err != nil {
+		return fmt.Errorf("service creation failed: %w", err)
```

Comment:
Use `errors` (instead of `github.com/pkg/errors`), and have `error.Is(…)` just `return nil`.